hw/ip/clkmgr/data/clkmgr.sv.tpl

```diff
@@ -84,11 +99,14 @@ module clkmgr import clkmgr_pkg::*; (
   ) u_${src['name']}_div (
     .clk_i(clk_${src['src']}_i),
     .rst_ni(rst_${src['src']}_ni),
+    .step_down_req_i(step_down_req == lc_ctrl_pkg::On),
+    .step_down_ack_o(step_down_acks[${loop.index}]),
     .test_en_i(scanmode_i),
     .clk_o(clk_${src['name']}_i)
   );
 % endfor
 
+  assign lc_clk_bypass_ack_o = &step_down_acks ? lc_ctrl_pkg::On : lc_ctrl_pkg::Off;
```

Comment:
Can you use a `prim_lc_sender` for this output? That module instantiates "hand-picked" flop cells.

Reply:
ah yes good point.